php-checks/src/test/resources/checks/CodeFollowingJumpStatementCheck.php

```diff
@@ -148,9 +148,10 @@ function functionIsDead() {
   function foo() {}// Noncompliant
 }
 
-function invalidJump() {
-  continue;
-}
+//No valid PHP - break the test case
+//function invalidJump() {
+//  continue; // no break/continue possible on level 1
+//}
```

Comment:
I don't understand why you commented out this function.
I don't see how how it "breaks the test case".
It seems that PHP 5.6 doesn't throw any syntax error on it, so I would prefer to keep this function here to ensure that we don't crash the analysis on such code.